packages/ra-no-code/src/ui/Ready.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { Ready as RaReady } from 'react-admin';
+import { ImportResourceDialog } from './ImportResourceDialog';
+
+export const Ready = () => (
+    <>
+        <RaReady />
```

Comment:
I think you'll need to design an entire ready page and get rid of the default one.

Reply:
It was the fastest way to have something. I planned to design something better in another PR